mix.exs

```diff
@@ -34,14 +34,17 @@ defmodule Livebook.MixProject do
 
   defp deps do
     [
-      {:phoenix, "~> 1.5.9"},
-      # We point LV to an exact version, because we install
-      # the npm package from there to bundle all the assets,
-      # so the Elixir-side version must match
-      {:phoenix_live_view, "0.16.0-dev", github: "phoenixframework/phoenix_live_view"},
-      {:phoenix_live_dashboard, "0.5.0-dev", github: "phoenixframework/phoenix_live_dashboard"},
+      # We point phoenix, phoenix_live_view and phoenix_tag to
```

Comment:
You mean phoenix_html instead of phoenix_tag. However, the question is: why are we using packages from npm? Shouldn’t we use the local ones?

Reply:
We are using local packages and they get bundled into `assets/priv`, then we want exactly the same Elixir (and consequently npm) packages to be installed, so that they match. Will clarify!